src/main/java/org/reactome/orthoinference/PathwaysInferrer.java

```diff
@@ -269,4 +277,8 @@ public static void setInferredEvent(Map<GKInstance,GKInstance> inferredEventCopy
 	{
 		inferredEventIdenticals = inferredEventCopy;
 	}
+
+	public static void setDiseaseInstance(GKInstance disease) {
+		diseasePathwayInst = disease;
+	}
```

Comment:
Is the ability to set the disease instance needed?  Could the logic from the `EventsInferrer` class be moved here so the disease instance is obtained directly instead of being changeable by a client?

```
private static GKInstance getDiseaseInstance() {
    if (diseasePathwayInst == null) {
        diseasePathwayInst = dbAdaptor.fetchInstance(InstanceUtilities.getDiseasePathwayDbId()); 
    }

    return diseasePathwayInst;
}
```